tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -418,3 +418,26 @@ export const restoreResourceTrashbin = async (
   const message = await page.locator(notificationMessageDialog).textContent()
   return message.trim().toLowerCase()
 }
+
+export interface searchResourceGlobalSearchArgs {
+  keyword: string
+  page: Page
+}
+
+export const searchResourceGlobalSearch = async (
+  args: searchResourceGlobalSearchArgs
+): Promise<void> => {
+  const { page, keyword } = args
+  await page.locator(globalSearchInput).fill(keyword)
+}
+
+export const getDisplayedResourcesFromSearch = async (page): Promise<string[]> => {
+  const result = await page.locator(searchList).allInnerTexts()
+  // the result has values like `test\n.txt` so remove new line
+  return result.map((result) => result.replace('\n', ''))
```

Comment:
Cool, you get every fileName and move it to array. 

What do you think about checking "path". maybe we should check path also? "PARENT/test-folder"   
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/84779829/188844177-d3555e26-8e9b-464c-b41e-666165ff5ca0.png">

Reply:
I'll try that

Reply:
@ScharfViktor I tried to do it through path but there's one problem. For the received shared the path is just `/`
Is this expected or a bug?
![resource-path](https://user-images.githubusercontent.com/41103328/189110647-634db73c-e473-450b-bfd7-3f2cfa1e4c88.png)

Reply:
I don't know why path is `/`. I build fresh web and it looks a little bit different. `data-test-resource-path` doesn't give us correct path
![Screenshot 2022-09-08 at 14 34 19](https://user-images.githubusercontent.com/84779829/189124241-e31f5e59-5f18-4454-9bf3-1119b84e6eb9.png)

Reply:
@ScharfViktor For me, it's not working for shares but `personal` it gives the correct path

![shared_withme](https://user-images.githubusercontent.com/41103328/189272684-db68c6c7-e36c-48f9-968a-7ee830150c58.png)

![personal](https://user-images.githubusercontent.com/41103328/189272690-44382d31-7e43-4e1d-92f6-a5250af40ba1.png)

Reply:
I can create a ticket for `data-test-resource-path` being `\` for shares

Reply:
created a ticket for it here https://github.com/owncloud/web/issues/7616

Reply:
@ScharfViktor so not use `data-test-resource-path` for now? Maybe we can use it after the issue has been solved?

Reply:
Sure, let's do that. Then we will check path later